about.html

```diff
@@ -0,0 +1,147 @@
+        The logo of Construction Engineering summit was selected among one's
+        that was in logo competition from 2nd to 6th june.
+      </p>
+
+      <div class="card cardsumittxt p-3 logo-wrapper">
+        <img
+          class="logoimg"
+          src="img/summitlogo.jpeg"
+          alt="logo"
+          width="100px"
+        />
+        <p cardsumit>
+          CES <br />
+          Construction Engineering <br />
+          summit 2020
+        </p>
+      </div>
+    </article>
+
+    <article class="card graduates p-lg-3 pb-lg-5 border-1 m-0">
+      <p class="graduates-header h2 text-center mt-4">
+        See the past CES Construction Engineering summit 2020
+      </p>
+      <h2 class="baseline text-center text-white fw-bold"></h2>
+      <p class="text-center mt-4">
+        take a look at the last two CC Global Summits which took place <br />
+        in Fiji and kigali Rwanda
+      </p>
+
+      <div
+        class="d-lg-flex my-5 px-5 justify-content-center justify-content-lg-center"
+      >
+        <div class="gradPhoto mt-4 me-lg-2 text-center align-items-center">
+          <p class="h1 year">2010</p>
+          <p class="below-year">CE Global Summit in amsterdam Netherlands</p>
+        </div>
+
+        <div class="gradPhotoes mt-4 ms-lg-2 text-center align-items-center">
+          <p class="h1 year">2015</p>
+          <p class="below-year">CE Global Summit in Kigali, Rwanda</p>
+        </div>
+      </div>
+    </article>
+    <article class="card partners mt-5 d-lg-none d-md-none bg-dark rounded-0">
+      <h2 class="partnertxt text-center mt-5">Partner</h2>
+      <h2 class="baseline fw-bold"></h2>
+      <div class="companies">
+        <div class="row d-flex justify-content-center mb-3">
+          <img class="col-4 py-4" src="icons/mozilla.svg" alt="" />
+          <img class="col-4" src="icons/google.svg" alt="" />
+          <img class="col-4" src="icons/naver.svg" alt="" />
+        </div>
+        <div class="row d-flex justify-content-center pb-4">
+          <img class="col-6" src="icons/daumkakao.svg" alt="" />
+          <img class="col-6" src="icons/airbnb.svg" alt="" />
+        </div>
+      </div>
+    </article>
+    <div class="reserved d-flex justify-content-center mt-4">
+      <img class="logo-end" src="img/summitlogo.jpeg" alt="" />
+      <p class="footertxt ende text-center">
+        <strong>2020 Global Construction Engineers<br /></strong>
+        Some Rights reserved.
+      </p>
+    </div>
+    <script
+      src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.11.6/dist/umd/popper.min.js"
+      integrity="sha384-oBqDVmMz9ATKxIep9tiCxS/Z9fNfEXiDAYTujMAeBAsjFuCZSmKbSSUnQlmh/jp3"
+      crossorigin="anonymous"
+    ></script>
+    <script
+      src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.min.js"
+      integrity="sha384-mQ93GR66B00ZXjt0YO5KlohRA5SY2XofN4zfuZxLkoj1gXtW8ANNCe9d5Y3eG5eD"
+      crossorigin="anonymous"
```

Comment:
For this section, let's stick to the Behance design by using the right background color for each section.
Your design:
![image](https://user-images.githubusercontent.com/60612329/226094444-c1a5fbef-b67e-4b72-b8d4-5aa6d5927d43.png)

Cindy's:
![image](https://user-images.githubusercontent.com/60612329/226094465-b3a37d06-35ae-4527-87fa-5ea0dcf71db1.png)

Reply:
Done